aws-ssm-document/src/main/java/com/amazonaws/ssm/document/DocumentModelTranslator.java

```diff
@@ -28,6 +28,10 @@
  * Translates CloudFormation's resource model to AWS resource model.
  */
 class DocumentModelTranslator {
+
+    private static final List<String> AWS_SSM_DOCUMENT_RESERVED_PREFIXES = ImmutableList.of(
+        "aws", "amazon", "amzn"
```

Comment:
Reserved prefix is "aws-", with the hyphen.

Reply:
updated